package.json

```diff
@@ -29,7 +29,7 @@
     "@types/angular": "^1.6.54",
     "@types/angular-animate": "^1.5.10",
     "@types/angular-mocks": "^1.7.0",
-    "@types/ckeditor": "^4.9.2",
+    "@types/ckeditor": "^4.9.8",
```

Comment:
Why is this `4.9.8`?

Reply:
NPM library page has that as its latest version, if I'm reading it right: https://www.npmjs.com/package/@types/ckeditor

Reply:
Yes, looks good. Thanks~